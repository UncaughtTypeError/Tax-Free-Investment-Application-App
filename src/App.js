import React, { useMemo } from 'react';
// Components
import Body from './containers/Body';
import Header from './presentational/Header';
import Wrapper from './presentational/Wrapper';
// Theme
import useMediaQuery from '@material-ui/core/useMediaQuery';
import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles';
import CssBaseline from '@material-ui/core/CssBaseline';

function App() {

  const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');

  const theme = useMemo(
      () =>
          createMuiTheme({
              palette: {
                  type: prefersDarkMode ? 'dark' : 'light',
                  primary: {
                      main: '#aa1111',
                  },
              },
          }),
      [prefersDarkMode],
  );

  return (
    <ThemeProvider theme={theme}>
      <CssBaseline />
      <Body>
        <Header />
        <Wrapper />
      </Body>
    </ThemeProvider>
  );
}

export default App;
