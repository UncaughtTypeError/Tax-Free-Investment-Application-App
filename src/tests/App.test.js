import React from 'react';
// Enzyme
import { shallow } from 'enzyme';
// Component
import App from '../App';

it('should render component tree', () => {
    expect(shallow(<App />)).toMatchSnapshot();
});