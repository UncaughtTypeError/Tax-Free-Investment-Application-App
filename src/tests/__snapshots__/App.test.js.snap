// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component tree 1`] = `ShallowWrapper {}`;
